blocks/components/editable/index.js

```diff
@@ -1,3 +1,98 @@
-export default function Editable( { value } ) {
-	return wp.element.createElement( 'p', null, value );
+const el = wp.element.createElement;
+let editorInstance = 1;
+
+export default class Editable extends wp.element.Component {
+	constructor( ...args ) {
+		super( ...args );
+		this.onInit = this.onInit.bind( this );
+		this.onSetup = this.onSetup.bind( this );
+		this.onChange = this.onChange.bind( this );
+		this.onFocusIn = this.onFocusIn.bind( this );
+		this.onFocusOut = this.onFocusOut.bind( this );
+		this.id = `tinymce-instance-${ editorInstance }`;
+		editorInstance++;
```

Comment:
This will be removed since I'm updating to Tiny4 which allows us to use the dom node instead of generating a random id  :)